### PR TITLE
Update max_message_size to reflect nap behavior change - https://docs.nginx.com/nginx-app-protect-waf/releases/release-notes-4.7/#resolved-issues

### PR DESCRIPTION
**Pull request by jnewfield**
### Proposed changes

Update max_message_size in example to be compatible with nap behavior change (after behavior change current example will result in NIC deployment failure) - https://docs.nginx.com/nginx-app-protect-waf/releases/release-notes-4.7/#resolved-issues

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [ ] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that all unit tests pass after adding my changes
- [ ] I have updated necessary documentation
- [ ] I have rebased my branch onto main
- [ ] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

File: docs/content/installation/integrations/app-protect-waf/configuration.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ For example, say you want to [log state changing requests](/nginx-app-protect-wa
     "content": {
         "format": "default",
         "max_request_size": "any",
-        "max_message_size": "5k"
+        "max_message_size": "64k"
```
Comment (vepatel) [inline docs/content/installation/integrations/app-protect-waf/configuration.md:126]:
```suggestion
      "max_request_size": "2k",
      "max_message_size": "5k"
```

File: docs/content/installation/integrations/app-protect-waf/configuration.md [hunk 2]
```diff
@@ -141,7 +141,7 @@ spec:
   content:
     format: default
     max_request_size: any
-    max_message_size: 5k
+    max_message_size: 64k
```
Comment (vepatel) [inline docs/content/installation/integrations/app-protect-waf/configuration.md:144]:
```suggestion
      "max_request_size": "2k",
      "max_message_size": "5k"
```

File: docs/content/installation/integrations/app-protect-waf/configuration.md (hunk 1 above)
Comment (danielnginx) [reply docs/content/installation/integrations/app-protect-waf/configuration.md:126]: Just to be clear that is the value suggested by NAP team 2k

File: docs/content/installation/integrations/app-protect-waf/configuration.md (hunk 2 above)
Comment (vepatel) [reply docs/content/installation/integrations/app-protect-waf/configuration.md:144]: nvm this value fails regex checks

**Review by ADubhlaoich — APPROVED** [review]
LGTM docs-wise

**Review by jjngx — APPROVED** [review]

**Pull request merged**